app/assets/javascripts/application.js

```diff
@@ -17,4 +17,5 @@
 //= require popper
 //= require list.min.js
 //= require bootstrap-sprockets
+//= require bootstrap/modal
```

Comment:
Is this require needed here? I think `bootstrap-sprockets` gives us all of the things?

Reply:
Yep looks like `bootstrap-sprockets` has what we need. I will remove the extra require.